server.js

```diff
@@ -0,0 +1,72 @@
+// Requirements
+const knex = require('knex');
+const express = require('express');
+const bcrypt = require('bcryptjs');
+const knexConfig = require('./knexfile');
+
+// Instantiations
+const server = express();
+const db = knex(knexConfig.development);
+
+// Middleware
+server.use(express.json());
+
+// Endpoints
+server.get('/', (req, res) => {
+    res.status(200).send('Server is running!');
+});
+
+server.post('/api/register', (req, res) => {
+    const creds = req.body;
+
+    const hash = bcrypt.hashSync(creds.password, 8);
+
+    creds.password = hash;
+
+    db('users_table')
+        .insert(creds)
+        .then(ids => {
+            const id = ids[0];
+
+            res.status(201).json(id);
+        })
+        .catch(err => {
+            console.log('/api/register POST error:', err);
+            res.status(500).send('Please try again later');
+        });
+});
+
+server.get('/api/users', (req, res) => {
+    db('users_table').select('id', 'username').then(users => {
+        res.status(201).json(users);
```

Comment:
In this endpoint, we are getting an array of all users in the `users_table`. We are not creating a new user, so I feel HTTP status code `200` (_OK/Success_) is more appropriate than `201` (_Created_).